### PR TITLE
Update Marathon UI version to 0.15-SNAPSHOT

### PR DESCRIPTION
**Pull request by philipnrmn**

### CODE REVIEW

File: project/build.scala [hunk 1]
```diff
@@ -299,7 +299,7 @@ object Dependency {
     val PlayJson = "2.4.3"
     val JsonSchemaValidator = "2.2.6"
     val RxScala = "0.25.0"
-    val MarathonUI = "0.14.1"
+    val MarathonUI = "0.15-SNAPSHOT"
```
Comment (gkleiman) [inline project/build.scala]: I think the version string should be `0.15.0-SNAPSHOT`.

Comment (philipnrmn) [inline project/build.scala]: d'oh

**Comment by philipnrmn** [comment]
Fixed it up. Thanks for the catch.

**Comment by gkleiman** [comment]
¡Muchas gracias! :rocket:

**Pull request merged**